module/Application/view/partial/admin.phtml

```diff
@@ -17,7 +17,7 @@ $flash = $this->plugin('FlashMessenger');
                 <ul class="nav navbar-nav">
                     <?php if (null === $this->identity()): ?>
                         <li>
-                            <a href="<?= $this->url('user') ?>">
```

Comment:
Another question. Why is this change necessary? Wouldn't it be a lot easier to just have implemented user_type member as a default in the route?

Reply:
I think I did it like this as the admin page is specifically for members. However, in the very same commit I made `member` the default route of `user_type` (see `User/config/module.config.php`), so I probably forgot to change it back (or I thought being explicit was better).